src/utils/date.ts

```diff
@@ -1,13 +1,89 @@
-const getDateDisplayValue = (date: string | number) => new Date(date).toISOString().split('T')[0];
+import { DateTime } from 'luxon';
 
-export const getTodaysDateFormatted = () => {
-  return getDateDisplayValue(Date.now());
+/**
+ * Helper util that returns default time zone as a fallback.
+ */
+const tz = (timeZoneId?: string) => timeZoneId || 'Etc/UTC';
+
+/**
+ * Parse a date string with zone implied.
+ * eg. '2023-01-10 in 'America/Los_Angeles'
+ */
+const getDateFromString = (date: any, timeZoneId?: string) => {
```

Comment:
Should this be `date: string`?

Reply:
Oh, I see that it is called by `getDate`, which explicitly accepts a `number` as well as a string. Could it be `string | number`?

Reply:
also, i realized these could be breaking changes - going to rename this file and restore the original version. We can use the correct file based on opt-in flag.